x-pack/plugins/secops/public/components/load_more_table/index.tsx

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiBasicTable,
+  EuiButton,
+  EuiButtonEmpty,
+  EuiContextMenuItem,
+  EuiContextMenuPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPopover,
+  EuiTitle,
+} from '@elastic/eui';
+import { isEmpty } from 'lodash/fp';
+import React from 'react';
+import styled from 'styled-components';
+
+import { EventItem } from '../../../common/graphql/types';
+import { LoadingPanel } from '../loading';
+
+export interface HoryzontalBarChartData {
+  x: number;
+  y: string;
+}
+
+export interface ItemsPerRow {
+  text: string;
+  numberOfRow: number;
+}
+
+interface BasicTableProps {
+  // tslint:disable-next-line:no-any
+  pageOfItems: any[];
+  columns: Columns[];
+  title: string | React.ReactNode;
+  loading: boolean;
+  loadingTitle?: string;
+  hasNextPage: boolean;
+  loadMore: () => void;
+  updateLimitPagination: (limit: number) => void;
+  itemsPerRow?: ItemsPerRow[];
+  limit: number;
+}
+
+interface BasicTableState {
+  isPopoverOpen: boolean;
+}
+
+export interface Columns {
+  field?: string;
+  name: string;
+  isMobileHeader?: boolean;
```

Comment:
Is this option used? I did a quick search and didn't see `isMobileHeader` being used.